src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidator.cs

```diff
@@ -71,5 +89,29 @@ public IEnumerable<ModelValidationResult> Validate(ModelValidationContext valida
 
             return Enumerable.Empty<ModelValidationResult>();
         }
+
+        private static IDictionary<Type, ValidationAttributeAdapterFactory> AttributeAdapters =
+            ValidationAttributeAdapterTable.AttributeFactories;
```

Comment:
Unnecessary. Would also be misplaced (should be at top of file w/ rest of the fields) if it was useful.